index.html

```diff
@@ -13,8 +13,22 @@
 
```

Comment:
remember when you have multiple nav components in a page to link all of them on each page, both in the header and the footer for easy navigation between pages